src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,98 @@
-export const Pagination = () => {};
+import { useCallback } from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  total: string[],
+  currentPage: number,
+  perPage: number,
+  onPageChange: (value: number) => void | number,
```

Comment:
Change the return type of `onPageChange` to `void` since the callback shouldn't return a value.